Detectors/GlobalTracking/src/MatchTOF.cxx

```diff
@@ -281,7 +281,7 @@ void MatchTOF::attachInputTrees()
   if (!mInputTreeTracks->GetBranch(mTracksBranchName.data())) {
     LOG(FATAL) << "Did not find tracks branch " << mTracksBranchName << " in the input tree";
   }
-  mInputTreeTracks->SetBranchAddress(mTracksBranchName.data(), &mTracksArrayInp);
+  mInputTreeTracks->SetBranchAddress(mTracksBranchName.data(), &mTracksArrayInpVect);
```

Comment:
just to let you know: for the ITS/TPC matching I've eliminated the support of running with the local reading from the tree, left the DPL version only. The code becomes increasingly messy with support of both ways.

Reply:
I missed this comment in my previous review and then I applied changes to mantain also this functionality. If you agree I would keep it working for the moment (for debugging). I will dismiss it in the future once I will not  be able to support it.

Reply:
Sure, your choice.